### PR TITLE
TreeSize-Free: Add version 4.3.1

### PR DESCRIPTION
**Pull request by TheZoker**
Add TreeSize Free:
https://www.jam-software.de/treesize_free/

### CODE REVIEW

File: bucket/treesize-free.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://www.jam-software.de/treesize_free/",
```
Comment (Ash258) [inline bucket/treesize-free.json]:
```suggestion
    "homepage": "https://www.jam-software.com/treesize_free",
```

File: bucket/treesize-free.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    "url": "https://downloads.jam-software.de/treesize_free/TreeSizeFree-Portable.zip",
+    "hash": "4CD6A7E1443DB8E45B0E1E042A7DD3A8884FC42407AE699DFB61087B84D4D47E",
+    "bin": "TreeSizeFree.exe",
+    "shortcuts": [
+        [
+            "TreeSizeFree.exe",
+            "TreeSizeFree"
+        ]
+    ]
```
Comment (Ash258) [inline bucket/treesize-free.json]:
```suggestion
    ],
    "checkver": {
        "url": "https://www.jam-software.com/treesize_free/changes.shtml",
        "regex": "Version\\s+([\\d.]+):"
    },
    "autoupdate": {
        "url": "http://downloads.jam-software.com/treesize_free/TreeSizeFree-Portable.zip",
    }
````

File: bucket/treesize-free.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://www.jam-software.de/treesize_free/",
+    "description": "Quickly capture directory sizes and memory guzzlers.",
+    "version": "4.3.1",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.jam-software.de/company/freeware_license.shtml"
+    },
+    "url": "https://downloads.jam-software.de/treesize_free/TreeSizeFree-Portable.zip",
```
Comment (Ash258) [inline bucket/treesize-free.json]:
```suggestion
    "url": "http://downloads.jam-software.com/treesize_free/TreeSizeFree-Portable.zip",
```

image

File: bucket/treesize-free.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "hash": "4CD6A7E1443DB8E45B0E1E042A7DD3A8884FC42407AE699DFB61087B84D4D47E",
+    "bin": "TreeSizeFree.exe",
+    "shortcuts": [
+        [
+            "TreeSizeFree.exe",
+            "TreeSizeFree"
```
Comment (Ash258) [inline bucket/treesize-free.json]:
```suggestion
            "TreeSize Free"
```

**Comment by TheZoker** [comment]
Again thanks for the feedback! Also fixes all the issues

**Comment by TheZoker** [comment]
@Ash258 Any news here?

**Pull request merged**